locations/spiders/denizbank.py

```diff
@@ -18,6 +18,7 @@ def parse(self, response):
             item = DictParser.parse(poi)
             # TODO: map "is-visually-imparied", "is-orthopedical-handicapped", "is-exchange-support" to OSM tags if possible
             item["ref"] = poi.get("code")
+            item["name"] = "Denizbank"
```

Comment:
Is there a reason the ```name``` field in the source data is no good to use? (the field should capture a unique name of each store, rather than the generic brand name that is already covered by ```item["brand"]```.

Reply:
Thanks for the question. I followed `name` tag [page from OSM wiki](https://wiki.openstreetmap.org/wiki/Key:name):

> As a rule of thumb, the primary name would be the most obvious name of the feature, the one that end users expect data consumers to expose in a label or other interface element. 

But I've just looked one more time into [DATA_FORMAT.md](https://github.com/alltheplaces/alltheplaces/blob/master/DATA_FORMAT.md), and I guess I can use `name` field from the source data.

Reply:
Yep, we don't universally use OSM tagging. Our documentation is a better place to start with. Would you be able to file a new PR to fix this please?

Reply:
Thanks! The change is made in https://github.com/alltheplaces/alltheplaces/pull/5328

Reply:
I think at some point we should move to OSMs `name` and `branch`, but maybe not just yet. https://github.com/alltheplaces/alltheplaces/issues/4772